support/mock/builder.go

```diff
@@ -36,7 +36,9 @@ func NewBuilder(ctx context.Context, receiver addr.Address) *RuntimeBuilder {
 		expectValidateCallerAddr: nil,
 		expectValidateCallerType: nil,
 
-		expectSends: make([]*expectedMessage, 0),
+		expectSends:   make([]*expectedMessage, 0),
+		actorCodeCIDs: make(map[addr.Address]cid.Cid),
+		newActorAddr:  addr.Undef,
```

Comment:
Move these last two up with the other fake values (rather than mock expectations), line ~33